src/langcheck/metrics/de/_translation.py

```diff
@@ -31,27 +31,30 @@ def _translate(self, texts: str) -> str:
         tokenization = self._translation_pipeline.tokenizer(
             texts, return_tensors="pt")  # type: ignore
         if tokenization.input_ids.shape[1] > (self._max_length / 2):
-            blocks = floor(
-                tokenization.input_ids.shape[1] / self._max_length) + 3
+            # Split the text into blocks, if it is too long
+            # adding 2 blocks to avoid problems with long texts
+            # NB: this comes from a few 100 tests, but it is not a science
```

Comment:
Can we update this comment now to reflect how we're computing the number of blocks?

Reply:
good point, done